man/chpl.txt

```diff
@@ -103,16 +103,18 @@ OPTIONS
   --[no-]dead-code-elimination   Enable [disable] dead code elimination.
 
   --fast            Turns off all runtime checks using --no-checks, turns
-                    on --no-ieee-float, -O and --specialize, and enables all
-                    compiler optimizations in the rest of this section.
+                    on -O and --specialize, and enables most other compiler
```

Comment:
I realize you've inherited this text, but I don't think of --fast as enabling the other optimizations in this section.  They're enabled by default and --fast doesn't affect them, aren't they?